sstables/sstable_directory.hh

```diff
@@ -161,7 +161,7 @@ private:
 
     template <typename Container, typename Func>
     requires std::is_invocable_r_v<future<>, Func, typename std::decay_t<Container>::value_type&>
-    future<> parallel_for_each_restricted(Container&& C, Func&& func);
+    future<> parallel_for_each_restricted(Container C, Func&& func);
```

Comment:
I don't see how this helps anything. The original declaration doesn't move anything by default, and allows passing  values and references.

Reply:
1. you're looking at an outdated version.
2. the original definition moved the container to `do_with(std::move(C)`